.travis.yml

```diff
@@ -62,6 +62,24 @@ matrix:
       script: bundle exec rake beaker
       services: docker
       bundler_args: --without development
+    - rvm: 2.5.1
+      dist: trusty
+      env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_debug=true BEAKER_setfile=debian8-64{hypervisor=docker\,hostname=debian8-64.example.com}
+      script: bundle exec rake beaker
+      services: docker
+      bundler_args: --without development
+    - rvm: 2.5.1
+      dist: trusty
+      env: PUPPET_INSTALL_TYPE=agent BEAKER_IS_PE=no BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_debug=true BEAKER_setfile=debian8-64{hypervisor=docker\,hostname=debian8-64.example.com}
+      script: bundle exec rake beaker
+      services: docker
+      bundler_args: --without development
+    - rvm: 2.5.1
```

Comment:
I do have my doubts about this one. Essentially we don't run any tests here but we do spend time to install it. I'm going to merge it and see about it in the modulesync PR.